src/deploy-tes-on-azure/Deployer.cs

```diff
@@ -327,12 +329,21 @@ public async Task<int> DeployAsync()
                             // Ensure all storage containers are created.
                             await CreateDefaultStorageContainersAsync(storageAccount);
 
+                            // Always place the compute nodes into the new batch subnet (needed for simplified communication with batch and is faster/cheaper for azure services access).
+                            await AssignMIAsNetworkContributorToResourceAsync(managedIdentity, resourceGroup);
+
                             if (string.IsNullOrWhiteSpace(settings["BatchNodesSubnetId"]))
                             {
                                 settings["BatchNodesSubnetId"] = await UpdateVnetWithBatchSubnet();
                             }
                         }
 
+                        if (installedVersion is null || installedVersion < new Version(4, 7))
+                        {
+                            // Storage account now requires Storage Blob Data Owner
```

Comment:
`Storage Blob Data Contributor` has "Read, write, and delete Azure Storage containers and blobs".  Just wanted to double check that Owner is really required

https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles